nixpkgs_review/utils.py

```diff
@@ -62,3 +63,11 @@ def current_system() -> str:
         text=True,
     )
     return system.stdout
+
+
+def nix_nom_tool() -> str:
+    "Return `nom` if found in $PATH"
+    if not shutil.which("nom"):
+        return "nix"
+    else:
+        return "nom"
```

Comment:
nitpick simplification:

```suggestion
    if shutil.which("nom"):
        return "nom"
    else:
        return "nix"
```